cronet/src/main/java/io/grpc/cronet/InternalCronetCallOptions.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2019 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.cronet;
+
+import io.grpc.CallOptions;
+import io.grpc.Internal;
+
+/**
+ * Internal accessor class for call options using with the Cronet transport. This is intended for
+ * usage internal to the gRPC team. If you *really* think you need to use this, contact the gRPC
+ * team first.
+ */
+@Internal
+public final class InternalCronetCallOptions {
+
+  /**
+   * @deprecated Use {@link #withAnnotation} instead.
+   */
+  @Deprecated
+  public static final CallOptions.Key<Object> CRONET_ANNOTATION_KEY =
```

Comment:
This probably shouldn't exist. Since you are migrating users anyway, they can swap to the withAnnotation implementation. But it's minor, so whatever.

Reply:
Done. Thanks for catching it.